aip/general/0203.md

```diff
@@ -189,16 +189,18 @@ an RPC or a resource in an [IaC][] client.
 ## History
 
 In 2023-05 field_behavior was made mandatory. Prior to this change, the
-annotation was often omitted. Its values, e.g. REQUIRED, OUTPUT_ONLY, and
-IMMUTABLE, are relied upon to produce high quality clients. Further, when the
-value is added after the fact or changes, within a major version, it is
-backwards-incompatible, which is likely to break clients.
+annotation was often omitted. Its values are relied upon to produce high quality
```

Comment:
Thanks! can you update line 185 while you're at it to clarify that no annotation -> optional is a backwards-compatible change? That's the line that I think raised the YAQS question in the first place.

Reply:
Done!